### PR TITLE
PIL-136 Repayment Guidance & amount

### PR DESCRIPTION
**Pull request by javed-iqbal1**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -162,13 +162,15 @@ features {
...
   asaAccessEnabled = true
+  repaymentsAccessEnabled = true
 
   welsh-translation = false
   incorporatedEntityBvEnabled = false
   partnershipBvEnabled = false
   privateBetaEnabled = false
   grsStubEnabled = true
   rfmAccessEnabled = true
+  requestRefundEnabled = true
```
Comment (jassalrichy) [inline conf/application.conf:173]: remove line 173 as not required (replaced with line 165 above)

File: app/config/FrontendAppConfig.scala [hunk 2]
```diff
@@ -109,9 +109,10 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
   val eacdHomePageUrl:              String  = configuration.get[String]("urls.eacdHomePage")
   val howToRegisterPlr2GuidanceUrl: String  = configuration.get[String]("urls.howToRegisterPlr2Guidance")
 
-  val rfmAccessEnabled:    Boolean     = configuration.get[Boolean]("features.rfmAccessEnabled")
-  def allowlistEnabled:    Boolean     = configuration.getOptional[Boolean]("filters.allowlist.enabled").getOrElse(false)
-  lazy val allowListedIps: Seq[String] = configuration.get[Seq[String]]("filters.allowlist.ips")
-  lazy val destination:    String      = configuration.get[String]("filters.allowlist.destination")
-  lazy val excludedPaths:  Seq[String] = configuration.get[Seq[String]]("filters.allowlist.excluded")
+  val rfmAccessEnabled:     Boolean     = configuration.get[Boolean]("features.rfmAccessEnabled")
+  val requestRefundEnabled: Boolean     = configuration.get[Boolean]("features.requestRefundEnabled")
```
Comment (jassalrichy) [inline app/config/FrontendAppConfig.scala:113]: line 113 is not required.

File: app/models/repayments/RefundAmount.scala [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.repayments
+
+import play.api.libs.json.{Json, OFormat}
+
+case class RefundAmount(amount: BigDecimal)
```
Comment (TevynAllen) [inline app/models/repayments/RefundAmount.scala:21]:
```suggestion
final case class RefundAmount(amount: BigDecimal)
```

File: test-utils/generators/Generators.scala [hunk 4]
```diff
@@ -129,4 +130,24 @@ trait Generators extends UserAnswersGenerator with PageGenerators with ModelGene
       Instant.ofEpochMilli(millis).atOffset(ZoneOffset.UTC).toLocalDate
     }
   }
+
+  def decimalInRangeWithCommas(min: Double, max: Double): Gen[String] = {
+    val numberGen = choose[Double](min, max)
+    genIntersperseString(numberGen.toString, ",")
+  }
+
+  private def arbitraryBigDecimalWithMax2DecimalPlaces: Gen[BigDecimal] =
```
Comment (TevynAllen) [inline test-utils/generators/Generators.scala:139]: private methods below public methods

**Review by TevynAllen — DISMISSED** [review]

**Review by AKKUM — APPROVED** [review]

**Review by AKKUM — APPROVED** [review]

**Pull request merged**